src/es6-module-bundler.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import generate from 'babel-generator';
+import * as babel from 'babel-types';
+import {Document} from 'polymer-analyzer';
+
+import {getAnalysisDocument} from './analyzer-utils';
+import {AssignedBundle, BundleManifest} from './bundle-manifest';
+import {Bundler} from './bundler';
+import {BundledDocument} from './document-collection';
+import {Es6Rewriter, getBundleModuleExportName, getModuleExportNames, hasDefaultModuleExport} from './es6-module-utils';
+import {ensureLeadingDot, stripUrlFileSearchAndHash} from './url-utils';
+
+export class Es6ModuleBundler {
```

Comment:
would it be clearer if this was just an exported function instead of a class? it seems like it's always instantiated, has one method called, and then thrown away.

Reply:
That's a valid point.  I've created two `bundle()` functions now exported from `es6-module-bundler.ts` and `html-bundler.ts`.  I got rid of the Es6ModuleBundler throw-away class but have kept the HtmlBundler class because I don't want to refactor that whole thing right now because it's unit tested etc, but I did mark it as deprecated in its description.

Reply:
As of commit 8b4050a432e16db515e5436a198c9093ab9a6861